SSH/legos/ssh_get_ec2_instances_with_low_available_disk_size/ssh_get_ec2_instances_with_low_available_disk_size.py

```diff
@@ -0,0 +1,65 @@
+##
+# Copyright (c) 2023 unSkript, Inc
+# All rights reserved.
+##
+import pprint
+from typing import List, Optional
+from pydantic import BaseModel, Field
+from unskript.legos.ssh.ssh_execute_remote_command.ssh_execute_remote_command import ssh_execute_remote_command
+
+
+class InputSchema(BaseModel):
+    hosts: List[str] = Field(
+        ...,
+        description='List of hosts to connect to. For eg. ["host1", "host2"].',
+        title='Hosts',
+    )
+    threshold: Optional[float] = Field(
+        default = 5, description='The disk size threshold in GB. Default- 5GB', title='Threshold(in GB)'
+    )
+    command: Optional[str] = Field(
+        default = "df -h /dev/xvda1", description='Command to get disk size. Default- df -h /dev/xvda1', title='Command to get disk size.'
+    )
+
+
+def ssh_get_ec2_instances_with_low_available_disk_size_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+def ssh_get_ec2_instances_with_low_available_disk_size(handle, hosts: list, threshold: float = 5, command:str = "df -h /dev/xvda1")-> List:
```

Comment:
Does the disk always get mounted at this path @shloka-bhalgat-unskript /dev/xvd1?

Reply:
No it does not. AWS has some combinations for device names given [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html) for linux. It can vary based on the os and the block device driver of the instance. For the Xen hypervisor (used by older EC2 instance types), the device names would often be in the /dev/xvdX format, like /dev/xvda or /dev/xvdf.

Reply:
Should I remove the default value?

Reply:
Do we need to automate this part based on the link you provided ?

Reply:
I've removed the parameter to get command. Now the code fetches the root disk name.